src/main/groovy/io/seqera/wave/auth/RegistryAuthServiceImpl.groovy

```diff
@@ -282,7 +282,7 @@ class RegistryAuthServiceImpl implements RegistryAuthService {
     void invalidateAuthorization(String image, RegistryAuth auth, RegistryCredentials creds) {
         final key = new CacheKey(image, auth, creds)
         cacheTokens.invalidate(key)
+        tokenStore.remove("key-" + key.stableKey())
```

Comment:
Let's turn `"key-" + key.stableKey()` is a small private method to better track its usage

Reply:
done